ts/parts-more/RadialAxis.ts

```diff
@@ -776,16 +805,20 @@ radialAxisMixin = {
             // prevent grid lines (or crosshairs, if enabled) from
             // rendering above the center after they supposed to be
             // displayed below the center point
-            if (!options.isCrosshair &&
-                (value as any < 0 || value as any > axis.height) && inverted) {
-                value = 0;
+            if (value) {
+                if (value < 0) {
+                    value = 0;
+                } else if (value > height) {
+                    value = isCrosshair ? height :
+                        (axis.reversed ? height : 0);
+                }
```

Comment:
Consider using `H.clamp(value, min, max)`

Reply:
Thanks!

Reply:
After some thinking I came to the conclusion that the `if else` statement here is not necessary (no matter if `value < 0` or `value > height`, it always must be 0 then).